### ISSUE TITLE
Add a checkbox to skip relauncher validation at each boot

### ISSUE THREAD

**Issue by boubou19** [GTNewHorizons/lwjgl3ify#147]
I think it would be cool to add a checkbox or something to tell relauncher that you want to skip the parameters screen, It has been a few times already that i launched my instance and went away during the booting time of NH just to come back and see it was waiting for my action to validate the parameters in the relauncher GUI.

**Comment by eigenraven** [comment]
It's planned and even coded in, I just don't want to show it before I have a way to force the window to show again after early crashes of the main game and with some sort of timeout dialog.
If you're bothered by it rn, there's a config for it already in `config/lwjgl3ify-relauncher.json`: `hideSettingsOnLaunch`

**Comment by boubou19** [comment]
oh nice, will use it, thank you!

**Comment by briermay** [comment]
I just updated to GTNH 2.6 and I cannot get rid of that annoying window every time I run it. I have searched and there is no config/lwjgl3ify-relauncher.json only config/lwjgl3ify.cfg and trying hideSettingsOnLaunch in that file in any location does absolutely nothing. how do I stop this stupidly annoying box every time I start the pack.

**Comment by chochem** [comment]
> I just updated to GTNH 2.6 and I cannot get rid of that annoying window every time I run it. I have searched and there is no config/lwjgl3ify-relauncher.json only config/lwjgl3ify.cfg and trying hideSettingsOnLaunch in that file in any location does absolutely nothing. how do I stop this stupidly annoying box every time I start the pack. I'm using Prism Launcher, so its absolutely useless to me anyhow as Prism manages all that on its own

if you are on prism that just means you installed it wrong. follow [this](https://gtnh.miraheze.org/wiki/Installing_and_Migrating)
with a correct install this does not pop up _at all_

**Comment by briermay** [comment]
I did all that. I still get that popup every time

**Comment by chochem** [comment]
there is no other way of saying this but: no, I am pretty sure you did not.

![image](https://github.com/GTNewHorizons/lwjgl3ify/assets/40274384/82e8f4ca-e852-4f80-bf44-a7a172809e6a)
this part in particular. maybe check by the help channel on the discord if you have more issues

**Comment by briermay** [comment]
1: don't tell me what I did or did not do your not here.
2: YES I did, but I do believe a file may have been locked and not copied over correctly. I fully rebooted the system, deleted the instance, and did everything again and its now working as expected.

Either way thank you for the help as it at least lead me down the path of why it was doing this. its not uncommon especially on windows for files to be held by the os even after exiting a program and not get copied.
